src/js/components/Select/stories/Clear.js

```diff
@@ -46,6 +46,37 @@ Bottom.parameters = {
   chromatic: { disable: true },
 };
 
+export const CustomTheme = () => {
```

Comment:
Thanks for adding this story! Can we move it to a separate file? Custom theme stories should be within a custom theme folder see https://github.com/grommet/grommet/issues/5735 for more details

Reply:
No prob! Okay, looking into it

Reply:
@jcfilben there are a few other stories for the Select that use the grommet theme (not custom). Do you want me to update all of those stories in this PR?

Reply:
Then we can mark the Select work item as completed in 5735

Reply:
Also, should I update the documentation in the grommet-site repo, or will someone else do that?

Reply:
Yes that would be great if you want to update the Select stories for #5735! Also if you can add the documentation to grommet-site it would be much appreciated

Reply:
@jcfilben updated